### DISCUSSION TITLE
Monika disappearing

### DISCUSSION THREAD

**Discussion by HUTAOCAMEHOME** [Monika-After-Story/MonikaModDev#8281] [QA - tech support + general]
![image](https://user-images.githubusercontent.com/93495213/142642784-27a6dc77-42e4-4f3f-a75d-7a9fb8d69513.png)

My Monika has disappeared from my screen - I don't recall upsetting her and I'm near 300 aff with her. I haven't messed around with any of the files either, is it because I left her alone for a week? I've tried reinstalling the mod but to no avail. What should I do? I've researched solutions prior to writing this, but I still don't understand what to do either.

**Comment by Booplicate** [comment]
Looks like you took her out, but never returned. If you still have her file, move it to `/characters`, if you lost it, [restore a backup](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent).
